index.html

```diff
@@ -69,11 +69,16 @@
 </head>
 
 <body class="d-flex flex-column min-vh-100">
-    <sgds-mainnav expand="never" mode="offcanvas">
+    <sgds-mainnav>
         <img width="130" src="https://www.designsystem.tech.gov.sg/assets/img/logo-sgds.svg" slot="brand" />
 
         <sgds-mainnav-item href="#">Home</sgds-mainnav-item>
         <sgds-mainnav-item href="#">About</sgds-mainnav-item>
+        <sgds-mainnav-item href="#" slot="end">Info</sgds-mainnav-item>
+        <sgds-mainnav-item href="#" slot="end">Contact Us</sgds-mainnav-item>
+        <sgds-mainnav-item href="#" slot="end">
```

Comment:
could we remove the `a.nav-link` wrapper if the slot contains a `button?`
The bottom border should not appear when mouse over a button.

![Screenshot 2022-10-14 at 10 55 09 AM](https://user-images.githubusercontent.com/6713735/195751763-2d4edc18-98e5-4678-91ae-32e3e75a5c51.png)